requirements-dev.txt

```diff
@@ -10,4 +10,5 @@ wheel
 flake8==3.9.2
 readme-renderer
 xdoctest
-pre-commit
\ No newline at end of file
+pre-commit
+requests==2.28.1
```

Comment:
Why are you pinning requests here?

Reply:
`requests` is already a requirement in setup.cfg so it is maybe not necessary to add it here. Just that I don't have the version pinned there. 
Was your intention to specifically pin the version or did you just want to add it in general?

Reply:
I'm not so good at Python packaging and I just added it because IDEA complained. I removed it again.